packages/integration-karma/test/wire/legacy-adapters/x/sameAdapterDifferentConfig/sameAdapterDifferentConfig.js

```diff
@@ -0,0 +1,23 @@
+import { LightningElement, wire, api } from 'lwc';
+import { EchoWireAdapter } from 'x/echoWireAdapter';
+import { FOO, BAR } from './constants';
+
+export default class SameAdapterDifferentConfig extends LightningElement {
+    @wire(EchoWireAdapter, { prop: FOO })
+    wireFuncFoo(data) {
+        this.wireFoo = data;
+    }
+
+    @wire(EchoWireAdapter, { prop: BAR })
+    wireFuncBar(data) {
+        this.wireBar = data;
+    }
```

Comment:
note that in the issue, the function name is wrongly set to `function` two times and that overrides the previous wire